openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/DataInsightSettings.spec.js

```diff
@@ -0,0 +1,245 @@
+/*
+ *  Copyright 2023 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { interceptURL, verifyResponseStatusCode } from '../../common/common';
+import { BASE_URL } from '../../constants/constants';
+
+const PIPELINE_NAME = 'cypress_dataInsight_pipeline';
+const REGION_NAME = 'US';
+
+describe('Data Insight settings page should work properly', () => {
+  beforeEach(() => {
+    cy.login();
+    interceptURL('GET', '/api/v1/teams/name/*', 'settingsPage');
+
+    cy.get('[data-testid="appbar-item-settings"]').click();
+    verifyResponseStatusCode('@settingsPage', 200);
+    cy.get('[data-testid="settings-left-panel"]').should('be.visible');
+
+    interceptURL(
+      'GET',
+      'api/v1/services/ingestionPipelines?fields=pipelineStatuses&service=OpenMetadata&pipelineType=dataInsight',
+      'ingestionPipelines'
+    );
+    interceptURL(
+      'GET',
+      '/api/v1/services/ingestionPipelines/OpenMetadata.OpenMetadata_dataInsight/pipelineStatus?startTs=*',
+      'pipelineStatus'
+    );
+
+    cy.get(`[data-menu-id*="openMetadata.dataInsight"]`)
+      .scrollIntoView()
+      .click();
+
+    verifyResponseStatusCode('@ingestionPipelines', 200);
+    verifyResponseStatusCode('@pipelineStatus', 200);
+  });
+
+  it('Add data insight pipeline should work properly', () => {
+    interceptURL(
+      'GET',
+      '/api/v1/services/metadataServices/name/OpenMetadata',
+      'serviceDetails'
+    );
+
+    cy.get('[data-testid="add-new-ingestion-button"]').click();
+
+    verifyResponseStatusCode('@serviceDetails', 200);
+
+    cy.get('[data-testid="name"]').clear().type(PIPELINE_NAME);
+
+    cy.get('[data-testid="next-button"]').click();
+
+    cy.get('#root\\/regionName').type(REGION_NAME);
+
+    cy.get('#root\\/useAwsCredentials').click({ waitForAnimations: true });
+
+    cy.get('#root\\/useAwsCredentials')
+      .invoke('attr', 'aria-checked')
+      .should('eq', 'true');
+
+    cy.get('[data-testid="next-button"]').click();
+
+    cy.intercept('POST', '/api/v1/services/ingestionPipelines').as(
+      'postIngestionPipeline'
+    );
```

Comment:
Use `interceptURL` here.

Reply:
@aniketkatkar97 have you addressed the comment?